### ISSUE TITLE
Accuracy on test set is {} based on the best model {} ?????????

### ISSUE THREAD

**Issue by lemo2012** [jiegzhan/multi-class-text-classification-cnn#20]
In you codes, the accuracy on test set is based on the latest model after all the train step finished, **not** based on the  best model saved in checkpoint. Am I right?

**Comment by papower1** [comment]
No. Please see step 6.2 in train.py

The variable ‘path’ is the path of best model.
Not the latest model.

We assume the least validation loss model is the best model.

The name of this method is ‘validation based early stopping’.
